contracts/BurnableStakeBank.sol

```diff
@@ -37,6 +37,7 @@ contract BurnableStakeBank is Lockable {
     Checkpoint[] public stakeHistory;
     Checkpoint[] public burnHistory;
     uint256 minimumStake;
+    address internal constant SYSTEM_ADDRESS = 0x00fffffffffffffffffffffffffffffffffffffffe;
```

Comment:
Should we just delete this line? I don't think its used in in this contract or in the inheritance hierarchy.

Reply:
Huh thought I just took this out.. Just committed these two changes.